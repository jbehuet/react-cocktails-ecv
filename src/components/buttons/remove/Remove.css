.remove {
    position: absolute;
    width: 56px;
    height: 56px;
    border-radius: 50%;
    background:#1f3254;
    color: #fff;
    top: 90px;
    left: 10%;
    border: 0;
    box-shadow: 0 2px 2px 0 rgba(0,0,0,0.14),0 3px 1px -2px rgba(0,0,0,0.12),0 1px 5px 0 rgba(0,0,0,0.2);
    cursor: pointer;
}