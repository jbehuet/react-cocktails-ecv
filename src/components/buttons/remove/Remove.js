import './Remove.css';

export function Remove({ onClick }) {
    return (
        <button className="remove" onClick={onClick}>
            <i className="fas fa-trash"></i>
        </button>
    )
}