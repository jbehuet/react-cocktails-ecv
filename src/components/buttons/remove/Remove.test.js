import React from "react";
import { create } from "react-test-renderer";

import { Remove } from "./Remove";

describe("Remove button component", () => {
  it("renders correctly", () => {
    const component = create(<Remove onClick={() => { }} />).toJSON();
    expect(component).toMatchSnapshot();
  });
});
