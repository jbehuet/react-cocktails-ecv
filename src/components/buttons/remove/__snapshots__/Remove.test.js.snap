// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Remove button component renders correctly 1`] = `
<button
  className="remove"
  onClick={[Function]}
>
  <i
    className="fas fa-trash"
  />
</button>
`;
